packages/components/src/TestComponent/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./TestComponent";
```

Comment:
This is a file generated from the script for the purpose of reviewing. TODO: remove this file before merging this PR.